### ISSUE TITLE
[Bitnami Wordpress] Services not starting automatically and not starting/restarting properly in command-line

### ISSUE THREAD

**Issue by alexgobert** [bitnami/vms#1738]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

5d450c08-3eb7-1892-bd67-9fc991a04815

### bndiagnostic output

===== Begin of bndiagnostic tool output =====

    ✓ Processes: No issues found
    ? Connectivity: Found possible issues
    ✓ Php: No issues found
    ? Apache: Found possible issues
    ✓ Resources: No issues found
    ✓ Wordpress: No issues found

[Connectivity]

Server ports 22, 80 and/or 443 are not publicly accessible. Please check the
following guide to open server ports for remote access:

https://docs.bitnami.com/general/faq/administration/use-firewall/

[Apache]

Found recent error or warning messages in the Apache error log.
Press [Enter] to continue:
```
[Fri Dec 13 23:10:57.547573 2024] [proxy_fcgi:error] [pid 799:tid 1057] [client
**ip_address**:39866] AH01079: failed to make connection to backend: www-fpm
 [Fri Dec 13 23:22:57.023125 2024] [proxy:error] [pid 4986:tid 5251] (2)No such
file or directory: AH02454: FCGI: attempt to connect to Unix domain socket
/opt/bitnami/php/var/run/www.sock (www-fpm:8000) failed
 [Fri Dec 13 23:22:57.023181 2024] [proxy_fcgi:error] [pid 4986:tid 5251]
[client **ip_address**:56806] AH01079: failed to make connection to backend:
www-fpm

```
Please check the following guide to troubleshoot server issues:

https://docs.bitnami.com/general/apps/wordpress/troubleshooting/debug-erro
rs-apache/

===== End of bndiagnostic tool output =====

### bndiagnostic was not useful. Could you please tell us why?

Does not address issue at hand.

### Describe your issue as much as you can

Running a Lightsail instance of Bitnami Wordpress. On startup, neither apache nor gonit are running, verified by `sudo /opt/bitnami/ctlscript.sh status` and `sudo gonit status`. When I click the reboot button in Lightsail, the instance restarts automatically. I do not believe this is normal. I must ssh into the instance on every start up to turn on gonit and apache using `sudo gonit` and `sudo /opt/bitnami/ctlscript.sh start apache`. Without any arguments after `start`, `ctlscript.sh` fails with a bitnami.service error. Below is output from `sudo systemctl status bitnami.service`

> `× bitnami.service - LSB: bitnami init script
     Loaded: loaded (/etc/init.d/bitnami; enabled; preset: enabled)
     Active: failed (Result: exit-code) since Fri 2024-12-13 23:26:56 CST; 9min ago
    Process: 5885 ExecStart=/etc/init.d/bitnami start (code=exited, status=1/FAILURE)
        CPU: 2.273s

> Dec 13 23:26:56 ip-REDACTED bitnami[5944]: 2024-12-14T05:26:56.325Z - error: Unable to perform start operation Export start for mariadb failed with exit code 1
Dec 13 23:26:56 ip-REDACTED bitnami[6023]: ## 2024-12-13 23:26:56-06:00 ## INFO ## Running /opt/bitnami/var/init/post-start/010_bitnami_agent_extra...
Dec 13 23:26:56 ip-REDACTED bitnami[6023]: ## 2024-12-13 23:26:56-06:00 ## INFO ## Running /opt/bitnami/var/init/post-start/020_bitnami_agent...
Dec 13 23:26:56 ip-REDACTED bitnami[6023]: ## 2024-12-13 23:26:56-06:00 ## INFO ## Running /opt/bitnami/var/init/post-start/030_update_welcome_file...
Dec 13 23:26:56 ip-REDACTED bitnami[6023]: ## 2024-12-13 23:26:56-06:00 ## INFO ## Running /opt/bitnami/var/init/post-start/040_bitnami_credentials_file...
Dec 13 23:26:56 ip-REDACTED bitnami[6023]: ## 2024-12-13 23:26:56-06:00 ## INFO ## Running /opt/bitnami/var/init/post-start/050_clean_metadata...
Dec 13 23:26:56 ip-REDACTED systemd[1]: bitnami.service: Control process exited, code=exited, status=1/FAILURE
Dec 13 23:26:56 ip-REDACTED systemd[1]: bitnami.service: Failed with result 'exit-code'.
Dec 13 23:26:56 ip-REDACTED systemd[1]: Failed to start bitnami.service - LSB: bitnami init script.
Dec 13 23:26:56 ip-REDACTED systemd[1]: bitnami.service: Consumed 2.273s CPU time.`

I tried to follow other issues with the same `LSB: bitnami init script` error, but I found little success.

Side note, I disabled mariadb using [this issue](https://github.com/bitnami/vms/issues/1038) because I am using AWS RDS as my database.

EDIT: Redacted private IP of my instance

**Comment by gongomgra** [comment]
@alexgobert once the `/etc/monit/conf.d/mariadb.conf` file has been renamed into `mariadb.conf.disabled`, reload Gonit and SystemD configs

```
sudo gonit reload
sudo systemctl daemon-reload
sudo gonit status
```

**Comment by alexgobert** [comment]
![image](https://github.com/user-attachments/assets/ab77913a-4cd5-48fd-900a-11514c8f0083)
![image](https://github.com/user-attachments/assets/ad59a7aa-f800-4075-b56b-eb70160eebd3)

Still no dice. See above screenshots for the outputs of those commands. When I clicked Lightsail's "Reboot" button, it instantly went back to running and my terminal refused to reconnect via SSH. When I hard stop and start the instance, gonit is not running. See below for output of `sudo gonit status` once the instance is running again.

**Comment by alexgobert** [comment]
> That's expected! You need to wait for the instance to run the startup process before the SSH services is available.
That's interesting. I have other instances on other accounts with the same software stack and rebooting always takes a minute or so; rebooting has never been instantaneous for me.

> Please note that services take some time to be up and ready. Did you run the "sudo gonit status" command after waiting some minutes?
Yes, I've tried waiting 10 or so minutes before, and the web server is not accessible until I SSH in and run `sudo gonit`

**Comment by alexgobert** [comment]
Apologies for the delayed response. Regarding the UI change, reboots on my other account take a couple minutes as expected. I am not convinced that it's a UI change. Do you have any other theories?

See below for output of `free -m`

![Image](https://github.com/user-attachments/assets/95313e92-4fb2-4776-b8b3-3822fba3abd8)

**Comment by gongomgra** [comment]
I don't think you are running out of memory according to those numbers, and I also see the `gonit status` in the previous screenshots was correct (only Apache and PHP-FPM were monitored). I'm afraid I can't think of any reason for the issues you are facing. Can you try with a brand new instance?

**Comment by alexgobert** [comment]
I did spin up another instance using a snapshot of the original one, same behavior. I have not tried with a fresh one.

**Comment by alexgobert** [comment]
Ran into some issues with AWS - not letting me create a fresh instance. Will update when resolved.
